internal/builder/_static/field_geo_processor.yml

```diff
@@ -0,0 +1,119 @@
+  name: client.geo.continent_name
+- external: ecs
+  name: client.geo.country_iso_code
+- external: ecs
+  name: client.geo.region_name
+- external: ecs
+  name: client.geo.region_iso_code
+- external: ecs
+  name: client.geo.city_name
+- external: ecs
+  name: client.geo.name
+- external: ecs
+  name: client.geo.location
+
+- external: ecs
+  name: destination.geo.continent_name
+- external: ecs
+  name: destination.geo.country_iso_code
+- external: ecs
+  name: destination.geo.region_name
+- external: ecs
+  name: destination.geo.region_iso_code
+- external: ecs
+  name: destination.geo.city_name
+- external: ecs
+  name: destination.geo.name
+- external: ecs
+  name: destination.geo.location
+
+- external: ecs
+  name: host.geo.continent_name
+- external: ecs
+  name: host.geo.country_iso_code
+- external: ecs
+  name: host.geo.region_name
+- external: ecs
+  name: host.geo.region_iso_code
+- external: ecs
+  name: host.geo.city_name
+- external: ecs
+  name: host.geo.name
+- external: ecs
+  name: host.geo.location
+
+- external: ecs
+  name: observer.geo.continent_name
+- external: ecs
+  name: observer.geo.country_iso_code
+- external: ecs
+  name: observer.geo.region_name
+- external: ecs
+  name: observer.geo.region_iso_code
+- external: ecs
+  name: observer.geo.city_name
+- external: ecs
+  name: observer.geo.name
+- external: ecs
+  name: observer.geo.location
+
+- external: ecs
+  name: server.geo.continent_name
+- external: ecs
+  name: server.geo.country_iso_code
+- external: ecs
+  name: server.geo.region_name
+- external: ecs
+  name: server.geo.region_iso_code
+- external: ecs
+  name: server.geo.city_name
+- external: ecs
+  name: server.geo.name
+- external: ecs
+  name: server.geo.location
+
+- external: ecs
+  name: source.geo.continent_name
+- external: ecs
+  name: source.geo.country_iso_code
+- external: ecs
+  name: source.geo.region_name
+- external: ecs
+  name: source.geo.region_iso_code
+- external: ecs
+  name: source.geo.city_name
+- external: ecs
+  name: source.geo.name
+- external: ecs
+  name: source.geo.location
+
+- external: ecs
+  name: threat.enrichments.indicator.geo.continent_name
+- external: ecs
+  name: threat.enrichments.indicator.geo.country_iso_code
+- external: ecs
+  name: threat.enrichments.indicator.geo.region_name
+- external: ecs
+  name: threat.enrichments.indicator.geo.region_iso_code
+- external: ecs
+  name: threat.enrichments.indicator.geo.city_name
+- external: ecs
+  name: threat.enrichments.indicator.geo.name
+- external: ecs
+  name: threat.enrichments.indicator.geo.location
+
+- external: ecs
+  name: threat.indicator.geo.continent_name
+- external: ecs
+  name: threat.indicator.geo.country_iso_code
+- external: ecs
+  name: threat.indicator.geo.region_name
+- external: ecs
+  name: threat.indicator.geo.region_iso_code
+- external: ecs
+  name: threat.indicator.geo.city_name
+- external: ecs
+  name: threat.indicator.geo.name
+- external: ecs
+  name: threat.indicator.geo.location
```

Comment:
Is there any reason why threat would need to be included in this? Especially threat.enrichments which would not exist in any datastream outside the ones generated by SIEM rules (alerts indices).